### PR TITLE
Fix: Add helpers directly to the site to allow cache

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [x] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

### CODE REVIEW

File: src/sonarwhal-theme/helper/index.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
-/* eslint-disable object-shorthand, prefer-template */
-const Handlebars = require('handlebars');
+const globby = require('globby');
+const path = require('path');
+
 const pagination = require('./pagination');
+
+const basePath = path.join(__dirname, '..');
+const helpersPath = path.join(__dirname, '..');
```
Comment (molant) [inline src/sonarwhal-theme/helper/index.js]: Why do we have 2 variables with the same value?

File: src/sonarwhal-theme/source/js/helpers/compare.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+/* eslint-env browser */
```
Comment (molant) [inline src/sonarwhal-theme/source/js/helpers/compare.js]: All this `ESLint` configurations should go in the `.estlintrc` file in the `sonarwhal-theme/source/js` folder

Comment (sarvaje) [reply src/sonarwhal-theme/source/js/helpers/compare.js]: ok

**Comment by sarvaje** [comment]
@molant done!

**Pull request merged**